Component/Cart/cart.css

```diff
@@ -0,0 +1,70 @@
+.cart-page{
+    margin: 8rem auto;
+}
+
+table{
+    width: 100%;
+    border-collapse: collapse;
+}
+
+.cart-info{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+th{
+    text-align: left;
+    padding: 0.5rem;
+    background: var(--btn-background);
+    color: var(--white-font-color);
+    font-weight: normal;
+    font-size: 1rem;
+}
+
+td{
+    padding: 1rem 0.5rem;
+}
+
+td input{
+    width: 3rem;
+    height: 2rem;
+    pad: 0.5rem;
```

Comment:
I guess, you wanted to use padding here.

Reply:
yes